services/motion/builtin/builtin_test.go

```diff
@@ -1756,8 +1806,8 @@ func TestMoveOnMapNew(t *testing.T) {
 		timeoutCtx, timeoutFn := context.WithTimeout(ctx, time.Second*5)
 		defer timeoutFn()
 		executionID, err := ms.(*builtIn).MoveOnMapNew(timeoutCtx, req)
-		test.That(t, err, test.ShouldBeNil)
-		test.That(t, executionID, test.ShouldNotResemble, uuid.Nil)
+		test.That(t, err, test.ShouldBeError, errors.New("no need to move, already within planDeviationMM: 2600.000000"))
```

Comment:
can we please add a test case that shows that we get a different error / no error if the plan deviation meters is manually specified viam motion config?

Reply:
> can we please add a test case that shows that we get a different error / no error if the plan deviation meters is manually specified viam motion config?

unsure what you mean by this, isn't this case implicitly covered under any success case we now have?